pkg/build/controller/build/build_controller.go

```diff
@@ -40,7 +40,6 @@ import (
 	buildlister "github.com/openshift/origin/pkg/build/generated/listers/build/internalversion"
 	buildgenerator "github.com/openshift/origin/pkg/build/generator"
 	buildutil "github.com/openshift/origin/pkg/build/util"
-	"github.com/openshift/origin/pkg/cmd/server/bootstrappolicy"
```

Comment:
@deads2k i'm removing a dependency on pkg/cmd here by making a copy of the BuilderServiceAccountName constant...  appropriate?

Reply:
(it was that or add *new* deps on pkg/cmd elsewhere in the build pkg and i assumed that would make you unhappy)